client/src/components/new-request/main-container.vue

```diff
@@ -5,49 +5,62 @@
       <v-col cols="auto" v-if="displayedComponent !== 'Success'">
         <button id="back-to-search-btn"
                 class="modal-activator pa-0 link-std-sans-ul"
-                @click="cancelAnalyzeName">
+                @click="cancelAndResetState">
           <span class="link-std-sans-ul">
             <v-icon class="ma-0 pa-0 mr-n1 mini-back-arrow">arrow_back_ios</v-icon>
             {{ editMode ? 'Return' : 'Start Search Over' }}
           </span>
         </button>
+        <countdown-timer :timerName="timerName" colorString="#1669bb" bgColorString="#efefef" />
```

Comment:
Our UX/UI designers have become careful about our colour usage, so if possible use named colours (or Material named colours) (or variables from theme.scss) instead of hex colours here. (Same reason we use constants in code.)

Reply:
Busy cranking out fixes, I can update that, just trying to get things working here. I need to do a css clean up in a couple areas related to payments. Noted regarding the colors. The colors I have used are the same ones you use for fonts and buttons.

Reply:
It's OK for now.

Note that we have tickets asking to update some of those text/font/button/icon styles.

So it will be a game of catch-up. Later!